docs/navigate.rst

```diff
@@ -24,7 +24,7 @@ View VIP table
 
 1. Click the Visibility tab, and then from the :guilabel:`Table` list, click the :guilabel:`VIP` table.
 
-2. From the :guilabel:`Partition` list, click the partition you're interested in.
+2. From the :guilabel:`Partition` list, click the partition you're interested in. **All partition** partition is supported from v2.9 onwards for VIP table.
```

Comment:
The **All partition** partition is supported for the VIP table in v2.9 and later.